app/services/audio/audio.ts

```diff
@@ -342,6 +342,26 @@ export class AudioSource implements IAudioSourceApi {
     this.UPDATE(this.audioService.fetchAudioSource(this.sourceId));
   }
 
+  setSyncOffset(syncOffset: number) {
+    this.source.getObsInput().syncOffset = AudioService.msToTimeSpec(syncOffset / 1000000);
+    this.UPDATE(this.audioService.fetchAudioSource(this.sourceId));
+  }
+
+  setMonitoringType(monitoringType: obs.EMonitoringType) {
+    this.source.getObsInput().monitoringType = monitoringType;
+    this.UPDATE(this.audioService.fetchAudioSource(this.sourceId));
+  }
+
+  setFlags(flags: number) {
+    this.source.getObsInput().flags = flags;
+    this.UPDATE(this.audioService.fetchAudioSource(this.sourceId));
+  }
+
+  setAudioMixers(audioMixers: number) {
+    this.source.getObsInput().audioMixers = audioMixers;
+    this.UPDATE(this.audioService.fetchAudioSource(this.sourceId));
+  }
+
```

Comment:
Actually you have no need to add these functions. You should use `setSettings` instead. You can look at `app\services\scenes-collections\nodes\sources.ts` for example

Reply:
Nice, didn't see this function.